ui/src/App.tsx

```diff
@@ -1,45 +1,45 @@
-import { Calendar, momentLocalizer } from "react-big-calendar";
+import React, { MouseEvent } from "react";
 import moment from "moment";
-import "react-big-calendar/lib/css/react-big-calendar.css";
 import { useState } from "react";
+import { Calendar, momentLocalizer } from "react-big-calendar";
+import "react-big-calendar/lib/css/react-big-calendar.css";
+import { CalendarEvent } from "./configs/CalendarEvent";
 import s from "./App.module.css";
 
-const localizer = momentLocalizer(moment);
-const currentHour = new Date().getHours();
-const currentMinute = new Date().getMinutes();
-const padNumberWith0 = (num) => num.toString().padStart(2, "0");
-const defaultStartTime = `${padNumberWith0(currentHour)}:${padNumberWith0(currentMinute)}`;
-
-Date.prototype.addHours = (h) => {
-  this.setTime(this.getTime() + h * 60 * 60 * 1000);
-  return this;
-};
+const currentHour: number = new Date().getHours();
+const currentMinute: number = new Date().getMinutes();
+const padNumberWith0: Function = (num: Number): String =>
+  num.toString().padStart(2, "0");
+const defaultStartTime: string = `${padNumberWith0(
```

Comment:
why is String sometimes capitalized and others not? does it make a difference? lets be consistent

Reply:
We found that using 'string' is preferred. We have made the updates.
https://stackoverflow.com/questions/14727044/what-is-the-difference-between-types-string-and-string